notes/0.13.10/clean-leave-roots.markdown

```diff
@@ -0,0 +1,11 @@
+[@pdalpra]: http://github.com/pdalpra
+[106]: https://github.com/sbt/sbt/issues/106
+[2136]: https://github.com/sbt/sbt/pull/2136
+
+### Fixes with compatibility implications
+
+### Improvements
+
+- clean task does not delete root folders anymore, only contents [#106][106]/[#2136][2136] by [@pdalpra][pdalpra]
```

Comment:
> There may be even more tickets around this.

I'll look for other tickets.

> ALSO, did you run a performance test at all to see what the impact is? We may already be screwed for clean on windows, but I'm curious if this is noticeable on other platforms.

The very basic tests I ran showed no noticeable difference, but I'll test on 'big' projects to see its impacts in the large.